packages/quantic/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle.css

```diff
@@ -31,18 +31,21 @@
   height: 48px;
 }
 
-.searchbox__clear-button {
-  height: 48px;
-}
-
 .searchbox__submit-button:focus {
   border: none;
   border-top-right-radius: 0.25rem;
   border-bottom-right-radius: 0.25rem;
 }
 
+.searchbox__clear-button-container {
+  height: 48px;
+  margin: inherit;
```

Comment:
this is because `slds-align_absolute-center` was adding a margin: auto to the container, which was causing the clear button to be centered when the searchbox expanded. Setting the margin to inherit insures the clear button stays on top with the submit button when the searchbox expands:


<img width="747" alt="image" src="https://github.com/coveo/ui-kit/assets/73316533/afc0dee8-3aca-4650-8e48-c15665298310">